src/shm_main.c

```diff
@@ -43,6 +44,38 @@ struct shm_item {
     char *name;
 };
 
+/**
+ * @brief Item holding information about active connections owned by this process.
+ */
+typedef struct _conn_list_entry conn_list_entry;
+struct _conn_list_entry {
+    conn_list_entry *_next;
+    sr_cid_t cid;
+    int cid_fd;
+};
```

Comment:
Please also put this together such as
```C
typedef struct {
    conn_list_entry *_next;
    sr_cid_t cid;
    int cid_fd;
} conn_list_entry;
```

Reply:
Done.  Note slight change needed for _next to have a defined type.

Reply:
Right, okay.